translations/ga/census_household_gb_nir_ga.po

```diff
@@ -2,4301 +2,6502 @@ msgid ""
+#. Answer option
+msgctxt "Who do you rent from?"
 msgid "Other"
 msgstr "Eile"
 
-#. answer-id: number-of-vehicles-answer
-msgctxt "Answer for: In total, how many cars or vans are owned, or available for use, by members of this household?"
+#. Answer option
+msgctxt "In total, how many cars or vans are owned, or available for use, by members of this household?"
 msgid "None"
 msgstr "Ceann ar bith"
 
-#. answer-id: number-of-vehicles-answer
-msgctxt "Answer for: In total, how many cars or vans are owned, or available for use, by members of this household?"
+#. Answer option
+msgctxt "In total, how many cars or vans are owned, or available for use, by members of this household?"
 msgid "1"
 msgstr "1"
 
-#. answer-id: number-of-vehicles-answer
-msgctxt "Answer for: In total, how many cars or vans are owned, or available for use, by members of this household?"
+#. Answer option
+msgctxt "In total, how many cars or vans are owned, or available for use, by members of this household?"
 msgid "2"
 msgstr "2"
 
-#. answer-id: number-of-vehicles-answer
-msgctxt "Answer for: In total, how many cars or vans are owned, or available for use, by members of this household?"
+#. Answer option
+msgctxt "In total, how many cars or vans are owned, or available for use, by members of this household?"
 msgid "3"
 msgstr "3"
 
-#. answer-id: number-of-vehicles-answer
-msgctxt "Answer for: In total, how many cars or vans are owned, or available for use, by members of this household?"
+#. Answer option
+msgctxt "In total, how many cars or vans are owned, or available for use, by members of this household?"
 msgid "4 or more"
 msgstr "4 nó níos mó"
 
-#. answer-id: number-of-vehicles-answer-other
-msgctxt "Answer for: In total, how many cars or vans are owned, or available for use, by members of this household?"
-msgid "Enter the number of cars or vans"
-msgstr "Cuir isteach méid na ngluaisteán nó na veaineanna"
-
-#. answer-id: proxy-answer
-msgctxt "Answer for: Are you <em>{person_name}?</em>"
+#. Answer option
+msgctxt "Are you <em>{person_name}?</em>"
 msgid "Yes, I am"
 msgstr "Is ea, Is mise"
 
-#. answer-id: proxy-answer
-msgctxt "Answer for: Are you <em>{person_name}?</em>"
+#. Answer option
+msgctxt "Are you <em>{person_name}?</em>"
 msgid "No, I am answering on their behalf"
 msgstr "Ní hea, tá mé ag freagairt ar a son"
 
-#. answer-id: confirm-date-of-birth-answer
-msgctxt "Answer for: You are {age_in_years} years old. Is this correct?"
-msgid "No, I need to change my date of birth"
-msgstr "Níl, caithfidh mé mo dháta breithe a athrú"
-
-#. answer-id: confirm-date-of-birth-answer
-msgctxt "Answer for: {person_name} is {age_in_years} years old. Is this correct?"
-msgid "No, I need to change their date of birth"
-msgstr "Níl, caithfidh mé a ndáta breithe a athrú"
-
-#. answer-id: sex-answer
-msgctxt "Answer for: What is your sex?"
+#. Answer option
+msgctxt "You are {age} old. Is this correct?"
+msgid "Yes, I am {age} old"
+msgstr "Tá mé {age} bliain d'aois"
```

Comment:
I think there is an issue here. Looking at previous translations "bliain" is a translation for year so we end up with "years year" in runner.
![image](https://user-images.githubusercontent.com/1754822/105322040-2f479180-5bc0-11eb-91db-16df696a524a.png)

Reply:
changed